src/Symfony/Component/Scheduler/DependencyInjection/AddScheduleMessengerPass.php

```diff
@@ -60,7 +60,7 @@ public function process(ContainerBuilder $container): void
                     $attribute = ($container->getReflectionClass($serviceDefinition->getClass())->getAttributes(AsCommand::class)[0] ?? null)?->newInstance();
                     $commandName = $attribute?->name ?? $serviceDefinition->getClass()::getDefaultName();
 
-                    $message = new Definition(RunCommandMessage::class, [$commandName.($tagAttributes['arguments'] ? " {$tagAttributes['arguments']}" : '')]);
+                    $message = new Definition(RunCommandMessage::class, [$commandName.(!empty($tagAttributes['arguments']) ? " {$tagAttributes['arguments']}" : '')]);
```

Comment:
```suggestion
                    $message = new Definition(RunCommandMessage::class, [$commandName.(($tagAttributes['arguments'] ?? null) ? " {$tagAttributes['arguments']}" : '')]);
```

as we avoid `empty()` were possible

Reply:
Suggestion applied and added test case for `'arguments => ''`, thank you.